src/EmployerRequestApprenticeTraining/SFA.DAS.EmployerRequestApprenticeTraining/Application/Queries/GetSettings/GetSettingsResult.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SFA.DAS.EmployerRequestApprenticeTraining.Application.Queries.GetSettings
+{
+    public class GetSettingsResult
+    {
+        public int ExpiryAfterMonths { get; set; }
+        public int RemovedAfterExpiryMonths { get; set; }
+    }
+}
```

Comment:
Should this also include ProviderRemovedAfterRequestedMonths which is included in the api code?

Reply:
No as this is the employer outer api there is no need for the provider setting